lib/src/phone_number.dart

```diff
@@ -64,8 +64,8 @@ class PhoneNumber {
       );
 
   /// formats the nsn, if no [isoCode] is provided the phone number region is used.
-  String formatNsn({IsoCode? isoCode, NsnFormat format = NsnFormat.national}) =>
-      PhoneNumberFormatter.formatNsn(nsn, isoCode ?? this.isoCode, format);
+  String formatNsn({IsoCode? isoCode, NsnFormat format = NsnFormat.national, bool showCountyCode = false}) =>
```

Comment:
Nsn stands for "national significant digits" and does not include the country code.

There should be two methods

 String formatNsn({IsoCode? isoCode, NsnFormat format = NsnFormat.national}) =>
      PhoneNumberFormatter.formatNsn(nsn, isoCode ?? this.isoCode, format);

String format({IsoCode? isoCode, NsnFormat format = NsnFormat.national}) => '+$countryCode ${formatNsn(isoCode, format)}'